tests/backup/backup_orphaned_delete_test.go

```diff
@@ -0,0 +1,234 @@
+			log.Infof(fmt.Sprintf("The list of user [%s] backups from px-admin %v", nonAdminUserName, userBackupNames))
+			for _, backupName := range userBackupNames {
+				backupUID, _ := Inst().Backup.GetBackupUID(ctx, backupName, orgID)
+				_, err = DeleteBackup(backupName, backupUID, orgID, adminCtx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying deletion backup [%s] of non-admin user [%s] from px-admin user", backupName, nonAdminUserName))
+				err = DeleteBackupAndWait(backupName, adminCtx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("waiting for backup [%s] deletion", backupName))
+			}
+		})
+
+		Step("Verifying deletion of backup schedule of non-admin user from px-admin", func() {
+			ctx, err := backup.GetNonAdminCtx(nonAdminUserName, commonPassword)
+			log.FailOnError(err, "Fetching non admin ctx")
+			adminCtx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching admin ctx")
+			userBackupScheduleNames, err := GetAllBackupSchedulesForUser(nonAdminUserName, commonPassword)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of fetching backups [%v] of user [%s] from px-admin user", userBackupScheduleNames, nonAdminUserName))
+			log.Infof(fmt.Sprintf("The list of user [%s] backups from px-admin %v", nonAdminUserName, userBackupScheduleNames))
+			for _, userBackupScheduleName := range userBackupScheduleNames {
+				log.InfoD(fmt.Sprintf("Verifying deletion of backup schedule [%s] of non-admin user [%s] from px-admin user", userBackupScheduleName, nonAdminUserName))
+				backupScheduleUid, err := GetScheduleUID(userBackupScheduleName, orgID, ctx)
+				log.FailOnError(err, fmt.Sprintf("Fetching schedule uid for shedule [%s]", userBackupScheduleName))
+				err = DeleteScheduleWithClusterRef(userBackupScheduleName, backupScheduleUid, SourceClusterName, srcClusterUid, orgID, adminCtx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of deleting backup scheudle [%s] of user [%s] from px-admin user", userBackupScheduleName, nonAdminUserName))
+			}
+		})
+
+		Step("Verifying updation of cluster of non-admin user from px-admin", func() {
+			adminCtx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-admin ctx")
+			srcClusterConfigPath, err := GetSourceClusterConfigPath()
+			log.FailOnError(err, "Fetching source clusterconfigpath")
+			_, err = UpdateCluster(SourceClusterName, srcClusterUid, srcClusterConfigPath, orgID, invalidCredName, invalidCloudCredUID, adminCtx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of updation of cluster [%v] of user [%s] from px-admin user", SourceClusterName, nonAdminUserName))
+			dstClusterConfigPath, err := GetDestinationClusterConfigPath()
+			log.FailOnError(err, "Fetching destination clusterconfigpath")
+			_, err = UpdateCluster(destinationClusterName, destClusterUid, dstClusterConfigPath, orgID, invalidCredName, invalidCloudCredUID, adminCtx)
```

Comment:
Here if the platform is vanilla which does not require cloud cred for cluster addition, this update will not make any difference?

Reply:
yes in vanilla cluster with px it doesn't make any difference, In both case deletion for backups go through.
This testcase is more applicable on CSI based backups which is dependent on cloud account.
Here from automation side we are just validating the update functionality is enabled from admin.